### PR TITLE
Replace Discord links in docs with links to Slack

### LINKED ISSUE #1983: Update docs to remove any mention of Discord and replace with Kedro's Slack org details
## Description
As we migrate away from Discord to use our own Slack org, we need to update the docs to remove all mention of Discord and point users to Slack instead. We should also consider adding a section to explain why the Slack org exists (for support) and compare with how to use Github discussions/Github issues for Kedro-project specific discussions.

### PR DESCRIPTION
**Pull request by BSGalvan**
Signed-off-by: Bharath Saiguhan <bharathsaiguhan@protonmail.com>

## Description
PR to replace mentions of Discord in the docs with links to the new Kedro Slack organization.

Note: Additional changes to be made once consensus is reached regarding where would be an appropriate location to place the rationale for the switch from Discord to Slack. More at the [relevant GitHub discussion](https://github.com/kedro-org/kedro/discussions/1986) for more.

## Development notes
All mentions (🤞🏽) of Discord in the docs were changed to Slack invite links (where there were links to the Discord server previously), or reworded to mention the Slack organization explicitly.

Changes were then confirmed by `make build-docs` (to see the changes visually) as well as with `make linkcheck` (to make sure links were _actually_ working).

## Checklist

- [X] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [X] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [X] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

Should close #1983 once additional changes are made.

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro/pull/1988"></a>

### CODE REVIEW

File: docs/source/faq/faq.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Frequently asked questions
 
 The following lists a set of questions that we have been asked about Kedro in the past. If you have a different
- question which isn't answered here, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Discord Server](https://discord.gg/akJDeVaxnB).
+ question which isn't answered here, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Slack organization](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```
Comment (merelcht) [inline docs/source/faq/faq.md:4]: We use UK English spelling in our docs, so to keep things consistent: 

```suggestion
 question which isn't answered here, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```

File: docs/source/faq/faq.md [hunk 2]
```diff
@@ -172,4 +172,4 @@ If you're an academic, Kedro can also help you, for example, as a tool to solve
 
 ## How can I get my question answered?
 
-If your question isn't answered above, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Discord Server](https://discord.gg/akJDeVaxnB).
+If your question isn't answered above, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Slack organization](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```
Comment (merelcht) [inline docs/source/faq/faq.md:175]:
```suggestion
If your question isn't answered above, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```

File: docs/source/get_started/install.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ You should see an ASCII art graphic and the Kedro version number: for example,
 
 ![](../meta/images/kedro_graphic.png)
 
-If you do not see the graphic displayed, or have any issues with your installation, see the [frequently asked questions](../faq/faq.md), check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Discord Server](https://discord.gg/akJDeVaxnB).
+If you do not see the graphic displayed, or have any issues with your installation, see the [frequently asked questions](../faq/faq.md), check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Slack organization](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```
Comment (merelcht) [inline docs/source/get_started/install.md:31]:
```suggestion
If you do not see the graphic displayed, or have any issues with your installation, see the [frequently asked questions](../faq/faq.md), check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```

File: docs/source/get_started/prerequisites.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Installation prerequisites
 
-- Kedro supports macOS, Linux and Windows (7 / 8 / 10 and Windows Server 2016+). If you encounter any problems on these platforms, please check the [frequently asked questions](../faq/faq.md), [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or the  [Discord Server](https://discord.gg/akJDeVaxnB).
+- Kedro supports macOS, Linux and Windows (7 / 8 / 10 and Windows Server 2016+). If you encounter any problems on these platforms, please check the [frequently asked questions](../faq/faq.md), [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or the [Slack organization](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```
Comment (merelcht) [inline docs/source/get_started/prerequisites.md:3]:
```suggestion
- Kedro supports macOS, Linux and Windows (7 / 8 / 10 and Windows Server 2016+). If you encounter any problems on these platforms, please check the [frequently asked questions](../faq/faq.md), [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```

File: docs/source/index.rst [hunk 5]
```diff
@@ -39,9 +39,9 @@ Welcome to Kedro's documentation!
     :target: https://kedro.readthedocs.io/
     :alt: Docs build status
 
-.. image:: https://img.shields.io/discord/778216384475693066.svg?color=7289da&label=Kedro%20Discord&logo=discord&style=flat-square
-    :target: https://discord.gg/akJDeVaxnB
-    :alt: Discord Server
+.. image:: https://img.shields.io/badge/slack-chat-blueviolet.svg?label=Kedro%20Slack&logo=slack&style=flat-square
+    :target: https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw
+    :alt: Kedro's Slack organization
```
Comment (merelcht) [inline docs/source/index.rst:44]:
```suggestion
    :alt: Kedro's Slack organisation
```

**Review by merelcht — COMMENTED** [review]
Thanks for taking this up @BSGalvan 😄  I left a small comment around UK vs US spelling.

**Comment by stichbury** [comment]
That is amazingly fast work -- thanks @BSGalvan -- nice PR. I'm happy to approve it.

**Review by stichbury — APPROVED** [review]
🌟 🏆 🌟

File: docs/source/contribution/contribute_to_kedro.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 We welcome any and all contributions to Kedro, at whatever level you can manage. For example, you could:
 
-- Join the community on [Discord](https://discord.gg/akJDeVaxnB)
+- Join the community on [Slack](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw)
 - Troubleshoot other users' questions or get answers to your own queries on [GitHub discussions](https://github.com/kedro-org/kedro/discussions)
```
Comment (stichbury) [inline docs/source/contribution/contribute_to_kedro.md:6]:
```suggestion
- Review Kedro's [GitHub isusses](https://github.com/kedro-org/kedro/discussions) or raise your own
- Start a conversation about the Kedro project on [GitHub discussions](https://github.com/kedro-org/kedro/discussions)
```

File: docs/source/contribution/technical_steering_committee.md [hunk 7]
```diff
@@ -23,7 +23,7 @@ In this section, we detail:
 - Guide the community to use the right channel:
   - [Github](https://github.com/kedro-org/kedro/) for feature requests and bug reports
   - [GitHub discussions](https://github.com/kedro-org/kedro/discussions)
```
Comment (stichbury) [inline docs/source/contribution/technical_steering_committee.md:25]:
```suggestion
  - [GitHub discussions](https://github.com/kedro-org/kedro/discussions) to discuss the Kedro project
```

File: docs/source/contribution/technical_steering_committee.md [hunk 8]
```diff
@@ -23,7 +23,7 @@ In this section, we detail:
...
-  - [Discord](https://discord.gg/akJDeVaxnB)
+  - [Slack](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw)
```
Comment (stichbury) [inline docs/source/contribution/technical_steering_committee.md:26]:
```suggestion
  - [Slack](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw) for questions and to support other users
```

File: docs/source/faq/faq.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
 # Frequently asked questions
 
 The following lists a set of questions that we have been asked about Kedro in the past. If you have a different
- question which isn't answered here, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Discord Server](https://discord.gg/akJDeVaxnB).
+ question which isn't answered here, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```
Comment (stichbury) [inline docs/source/faq/faq.md:4]:
```suggestion
 question which isn't answered here, check out the [searchable archive from our retired Discord server](https://linen-discord.kedro.org/community or post a new query on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```

File: docs/source/faq/faq.md [hunk 10]
```diff
@@ -172,4 +172,4 @@ If you're an academic, Kedro can also help you, for example, as a tool to solve
 
 ## How can I get my question answered?
 
-If your question isn't answered above, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Discord Server](https://discord.gg/akJDeVaxnB).
+If your question isn't answered above, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```
Comment (stichbury) [inline docs/source/faq/faq.md:175]:
```suggestion
If your question isn't answered above, check out the [searchable archive from our retired Discord server](https://linen-discord.kedro.org/community or post a new query on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```

File: docs/source/get_started/install.md [hunk 11]
```diff
@@ -28,7 +28,7 @@ You should see an ASCII art graphic and the Kedro version number: for example,
 
 ![](../meta/images/kedro_graphic.png)
 
-If you do not see the graphic displayed, or have any issues with your installation, see the [frequently asked questions](../faq/faq.md), check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Discord Server](https://discord.gg/akJDeVaxnB).
+If you do not see the graphic displayed, or have any issues with your installation, see the [frequently asked questions](../faq/faq.md), check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```
Comment (stichbury) [inline docs/source/get_started/install.md:31]:
```suggestion
If you do not see the graphic displayed, or have any issues with your installation, see the [frequently asked questions](../faq/faq.md), check out the [searchable archive from our retired Discord server](https://linen-discord.kedro.org/community or post a new query on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw)..
```

File: docs/source/get_started/prerequisites.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
 # Installation prerequisites
 
-- Kedro supports macOS, Linux and Windows (7 / 8 / 10 and Windows Server 2016+). If you encounter any problems on these platforms, please check the [frequently asked questions](../faq/faq.md), [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or the  [Discord Server](https://discord.gg/akJDeVaxnB).
+- Kedro supports macOS, Linux and Windows (7 / 8 / 10 and Windows Server 2016+). If you encounter any problems on these platforms, please check the [frequently asked questions](../faq/faq.md), [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```
Comment (stichbury) [inline docs/source/get_started/prerequisites.md:3]:
```suggestion
- Kedro supports macOS, Linux and Windows (7 / 8 / 10 and Windows Server 2016+). If you encounter any problems on these platforms, please check the [frequently asked questions](../faq/faq.md), the [searchable archive from our retired Discord server](https://linen-discord.kedro.org/community or post a new query on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw)..
```

**Review by stichbury — APPROVED** [review]
Nice changes, thank you. I've made some suggestions to extend them slightly based on [the post I made last week ](https://github.com/kedro-org/kedro/discussions/1986) which aims to clarify which platform to use for support, issues and general discussions. So I've pointed people more at the Slack org (or the archive on linen) and directed away from github discussions except for more general conversation about the project.

I hope that makes sense, sorry not to mention it previously, it was all work in progress!

File: docs/source/faq/faq.md (hunk 9 above)
Comment (BSGalvan) [reply docs/source/faq/faq.md]: Here, the link to the linen archive above seems to return a 404. Maybe you meant [https://linen-discord.kedro.org](https://linen-discord.kedro.org) instead?

Comment (stichbury) [reply docs/source/faq/faq.md]: I did, thanks!

Comment (stichbury) [reply docs/source/faq/faq.md]: @BSGalvan I've now fixed that link throughout my suggestions where it arose, so all should work as expected.

**Comment by stichbury** [comment]
Hi @BSGalvan Would you be OK if I merge my changes to your commits into the branch, rebuild and merge to `main`? I'd like to get this change in if you're done with your updates as it's a fantastic PR and would be great to complete it.

**Comment by BSGalvan** [comment]
Yup, that sounds good to me! You can go ahead with that 👍🏼

**Review by yetudada — APPROVED** [review]
This is great! Thank you so much @BSGalvan!

**Pull request merged**